modules/infra/submodules/storage/variables.tf

```diff
@@ -148,11 +148,6 @@ variable "storage" {
     error_message = "Invalid 'deployment_type' for netapp filesystem, supported deployment types are 'MULTI_AZ_1', 'MULTI_AZ_2', 'SINGLE_AZ_1', and 'SINGLE_AZ_2'."
   }
 
-  validation {
```

Comment:
Removing validation which avoids creating netapp when storage is efs, we need this for migration